scripts/observers/start.py

```diff
@@ -40,10 +52,12 @@ def main():
     print("staring observers and indexers....")
 
     start_seed_node(working_dir)
-    start_observer(0, working_dir, 0)
-    start_observer(1, working_dir, 1)
-    start_observer(2, working_dir, 2)
-    start_observer(METACHAIN, working_dir, 3)
+    start_proxy(working_dir)
+    start_observer_and_indexer(METACHAIN, working_dir, 0)
+
+    num_of_shards = int(os.getenv('NUM_OF_SHARDS'))
```

Comment:
might add a check for num shards greater than 3

Reply:
added extra checks in the `start.py` file and `config.py` file